packages/core/admin/package.json

```diff
@@ -57,7 +56,6 @@
     "@strapi/typescript-utils": "4.11.4",
     "@strapi/utils": "4.11.4",
     "axios": "1.4.0",
-    "babel-loader": "^9.1.2",
     "babel-plugin-styled-components": "2.1.1",
```

Comment:
I think you are right for this one (will remove), but I'm afraid we can not remove more because `eslint` depends on all other babel dependencies.

Reply:
But they can be removed from the `admin` package right? cause eslint runs uses the deps from the root of the project iirc which has:

```
    "@babel/core": "^7.20.12",
    "@babel/eslint-parser": "^7.19.1",
    "@babel/preset-react": "7.18.6",
```

listed

Reply:
That's a good point, actually. Let me try that, but I think you are right.

Reply:
You are right. We don't expose any eslint extension point from admin and therefore we only need the top-level dependencies for eslint to work. Great catch! This will make the admin package much lighter in terms of installation size!

Reply:
Done.

Reply:
![Michael Scott Wink GIF](https://media2.giphy.com/media/ui1hpJSyBDWlG/200.gif)